src/util/text-utils.cc

```diff
@@ -191,14 +191,16 @@ inline bool is_nan_text(const std::string &in, const std::string &prefix) {
 template<typename T>
 bool convert_special_number(const std::string &str, T *out) {
```

Comment:
Please name this according to the Google style guide, i.e. ConvertSpecialNumber, and put 'static' before bool so that it's clear it's a local function.